haiku-apps/bescreencapture/bescreencapture-2.3.2.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
+COPYRIGHT="2014-2018 Stefano Ceccherini"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="c38d0860e97ab2b4070a0d62c0ea554e09328601f253d6c46b87dede7485830f"
+SOURCE_FILENAME="bescreencapture-${portVersion}.tar.gz"
+SOURCE_DIR="bescreencapture-${portVersion}"
```

Comment:
We don't need to define `SOURCE_FILENAME` because haikuporter already sets it to a default value, which is what we have here. So you can drop it. (Well, you can also keep it, but it's nicer to drop it :)
```
-SOURCE_FILENAME="bescreencapture-${portVersion}.tar.gz"
-SOURCE_DIR="bescreencapture-${portVersion}"
+SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
```
(For the curly braces, that's the same remark: we can keep them, or drop them. Please feel free to do whatever you prefer.)

Regarding the removal of the recipe for version `2.32`, as well as for the changes suggested in this review, I guess you know how to update the PR. But please feel free to ask if you need help.

Reply:
What about SOURCE-DIR ?
Is it needed ? I see you removed SOURCE_DIR and kept SOURCE_FILENAME instead.

Reply:
Thanks for the update!

* `SOURCE_DIR` is not required for this recipe because the name of the directory at the top level of the tarball is `bescreencapture-2.3.2`, and this is exactly what haikuporter uses as the default value for `SOURCE_DIR`.
In other words, we can drop the `SOURCE_DIR`, or keep it. Both would work fine.
(But we prefer to drop it, since it is not required.)

* Regarding the `SOURCE_FILENAME`, although the recipe works without it, we prefer to have it, so that haikuporter will rename the tarball to something that looks nicer than just `v2.3.2.tar.gz`.

So once you have dropped the `SOURCE_DIR` and added the `SOURCE_FILENAME`:
```diff
-SOURCE_DIR="bescreencapture-${portVersion}"
+SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
```
the PR will be ready for a merge :)

Reply:
A note on why SOURCE_FILENAME is important: our builders can store a mirror of all sources we use when they build things, for future reference. It is much easier to handle that mirror if sourcefiles are properly named, and it also may avoid name conflicts (which will happen sooner or later if only version numbers are used to name the source archives).